pkgs/development/python-modules/pyro-ppl/default.nix

```diff
@@ -33,7 +34,7 @@ buildPythonPackage rec {
     description = "A Python library for probabilistic modeling and inference";
     homepage = "http://pyro.ai";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ teh ];
-    broken = true;
+    platforms = lib.platforms.unix;
```

Comment:
`buildPythonPackage` will add this for you
```suggestion
```

Reply:
Is this only required if the package is linux or darwin only?

Reply:
removed again